zerver/actions/invites.py

```diff
@@ -476,6 +476,20 @@ def do_revoke_multi_use_invite(multiuse_invite: MultiuseInvite) -> None:
     notify_invites_changed(realm)
 
 
+def do_edit_multiuse_invite_link(
```

Comment:
I am not sure whether we require a `transaction.atomic` here since there is only a single database operation here. Let's keep this as it is for now.